compiler/testSuite/Vintop02.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/io;
+
+public function main() {
+  printBoolean(7 == 8); // @output 0
```

Comment:
@jclark this fails with error `error: intersection of operands of operator '==' is empty` but `7 + 1 == 8` does not.

Reply:
In that case this file should have a name starting with with `F`

Reply:
Fixed.

Reply:
> In that case this file should have a name starting with with `F`

Fixed.